build/templates/validate-commits.yaml

```diff
@@ -0,0 +1,11 @@
+# This template is used to validate that the commit messages follow the Conventional Commits specification (https://www.conventionalcommits.org/en/v1.0.0/).
+# Consider placing this at the beginning of the build pipeline to ensure that the commits are valid before proceeding with longer build steps.
+steps:
+  - task: CommitMessageValidator@0
```

Comment:
0 is not a valid version for the task. We can see that it fails in the Azure pipeline.